config/v1/types_cluster_version.go

```diff
@@ -806,6 +845,15 @@ type ConditionalUpdate struct {
 // for not recommending a conditional update.
 // +k8s:deepcopy-gen=true
 type ConditionalUpdateRisk struct {
+	// conditions represents the observations of the conditional update
+	// risk's current status. Known types are:
+	// * Applied, for whether the risk is applied to the current cluster.
```

Comment:
nit: I'd prefer "Applies" over "Applied".  In reality, any condition is going to be slightly stale, e.g. the ClusterVersion `status.conditions` `Failing` isn't actually "is failing this way right now" it's "was failing that way recently".  But we don't give consumers a way to determine freshness, and we expect them to act as if the gap between "how fresh the data actually is" and "right now" is almost always negligibly small.  To me, `Applied` sets folks up to worry about how stale the data is, and without a way to determine freshness, it's hard to be productive with that worry.  While `Applies` says "it's ok if you assume this is current; if we fall too far behind there will probably be other alerting".